solutions/swb-reference/integration-tests/tests/isolated/awsAccounts/update.test.ts

```diff
@@ -0,0 +1,62 @@
+/*
+ *  Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *  SPDX-License-Identifier: Apache-2.0
+ */
+import { resourceTypeToKey } from '@aws/workbench-core-base';
+import ClientSession from '../../../support/clientSession';
+import Account from '../../../support/resources/accounts/account';
+import Setup from '../../../support/setup';
+import HttpError from '../../../support/utils/HttpError';
+import { checkHttpError } from '../../../support/utils/utilities';
+
+describe('awsAccounts update negative tests', () => {
+  const setup: Setup = new Setup();
+  let adminSession: ClientSession;
+
+  beforeEach(() => {
+    expect.hasAssertions();
+  });
+
+  beforeAll(async () => {
+    adminSession = await setup.getDefaultAdminSession();
+  });
+
+  afterAll(async () => {
+    await setup.cleanup();
+  });
+
+  describe('when updating an account', () => {
+    let account: Account;
+
+    beforeEach(() => {
+      const accountId = `${resourceTypeToKey.account.toLowerCase()}-00000000-0000-0000-0000-000000000000`;
+      account = adminSession.resources.accounts.account(accountId);
+    });
+
+    describe('and the update params are invalid', () => {
+      test('it throws a validation error', async () => {
+        try {
+          const surpriseIntValue = 1 as unknown as string;
+
+          await account.update({
+            name: surpriseIntValue,
```

Comment:
Why can't you send `{ name: '1' }` ? Wasn't the signature `Record<string, unknown>`?

Reply:
the type for `account.update()` is `Record<string, string>`, which is what we want since thats what the expected shape of the request is _right now_. Probably want to change this to a real request object though, but given the other outstanding work my initial instinct is to leave it as code clean up for later. I could be swayed though if you think differently?